Resources/Prototypes/_NF/Entities/Objects/Tools/emag.yml

```
@@ -1,3 +1,19 @@
+- type: entity
+  parent: EmagUnlimited
+  id: EmagPass
+  name: cryptographic sequencer pass
+  description: A portable, disposable device hacking solution made by the Syndicate.
```

```suggestion
  description: A disposable card made by the Syndicate to hack devices. Works three times.
```
Blackknight's right, the description is awkward.